### PR TITLE
[search] Added "category" bit to trace parses.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

File: search/tracer.cpp [hunk 1]
```diff
@@ -35,7 +36,7 @@ string DebugPrint(Tracer::Parse const & parse)
   using TokenType = Tracer::Parse::TokenType;
 
   ostringstream os;
-  os << "Parse [";
+  os << "Parse [parse: ";
```
Comment (mpimenov) [inline search/tracer.cpp]: Вот это "parse", может, и лишнее. И так уже много двоеточий.

**Comment by mapsmetest** [comment]
Build triggered. sha1 is merged.

**Comment by mapsmetest** [comment]
Build started sha1 is merged.

**Comment by mapsmetest** [comment]
GitHub Watchdog Failure http://angola.mapsme.mail.msk:8080/job/GitHubWatchCat/
 No test results found.

**Pull request merged**